### PR TITLE
[DA-1162] preparing config and tooling for new ptsc test 1 env

### PR DESCRIPTION
**Pull request by rennat**
This is the new configuration and tool changes necessary to set up the new `all-of-us-rdr-ptsc-1-test` environment.

Also included is a `doc/creating_an_rdr_environment.md` file that documents the process.

### CODE REVIEW

File: doc/creating_an_rdr_environment.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    * biobank samples
+    * ghost id
+    * consent pdf
+0. Update config and tooling files
+    0. Create `rest-api/config/config_<PROJECT>.json` for the new project
+    0. Create `rest-api/cron_<PROJECT>.yaml` to override any settings from `cron_default.yaml` (if needed)
+    0. Update `rest-api/tools/auth_setup.sh` to know how to handle the new project
+    0. Update `rest-api/tools/deploy_app.sh` to know how to handle the new project
+    0. Update `rest-api/tools/build_cron_yaml.py` to know how to handle the new project
+0. From Google Cloud Console, Create new App Engine application
```
Comment (rennat) [inline doc/creating_an_rdr_environment.md]: The work done so far stops here. We need to confirm the config is correct for the new env before proceeding further.

File: rest-api/config/config_test_ptsc_1.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "vibrent@pmi-ptc-drc-test-harness.iam.gserviceaccount.com": {
+      "roles": ["ptc", "healthpro"],
+      "whitelisted_ip_ranges": {
+        "ip6": ["::0/0"],
+        "ip4": ["0.0.0.0/0"]
+      }
+    },
+    "circle-deploy@all-of-us-rdr-staging.iam.gserviceaccount.com": {
+      "roles": ["ptc", "healthpro", "rdr"]
+    },
```
Comment (rennat) [inline rest-api/config/config_test_ptsc_1.json]:
```suggestion
    }
```

File: rest-api/config/config_test_ptsc_1.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
+{
+  "allow_nonprod_requests": true,
+  "biobank_id_prefix": ["Y"],
```
Comment (rennat) [inline rest-api/config/config_test_ptsc_1.json]: Invent a new biobank prefix for each environment?

@Seeker1911

Comment (Seeker1911) [reply rest-api/config/config_test_ptsc_1.json]: I think just use the test prefix here, so PTSC doesn't need to adjust configs.

File: rest-api/tools/setup_database.sh [hunk 4]
```diff
@@ -112,22 +136,23 @@ if [ "${UPDATE_PASSWORDS}" = "Y" ] || [ "${CREATE_INSTANCE}" = "Y" ]
...
 		   cat tools/grant_permissions.sql | envsubst >> $UPDATE_DB_FILE
 		done
 	fi
 
+    echo "Queueing password change commands"
+    cat tools/update_passwords.sql | envsubst >> $UPDATE_DB_FILE
 
-	mysql -u "$ROOT_DB_USER" -p"$ROOT_PASSWORD" --host 127.0.0.1 --port ${PORT} < ${UPDATE_DB_FILE}
+	echo "applying database changes..."
+	FAILURE_MESSAGE=
```
Comment (rennat) [inline rest-api/tools/setup_database.sh]:
```suggestion
```

**Comment by rennat** [comment]
These codacy issues can be ignored. We need to disable that particular check

**Comment by Seeker1911** [comment]
we could use something else like grep instead of cat. Cat is slow compared to other methods.

**Pull request merged**